.github/workflows/issues.yml

```diff
@@ -0,0 +1,20 @@
+name: Add issues to DevProd project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          # You can target a repository in a different organization
+          # to the issue
+          project-url: https://github.com/orgs/cloudflare/projects/1
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Worth adding also something that kicks a project off to the Pages project if the `pages` label is added?

Reply:
Is there a Pages issue template? Could it auto-assign such a label?

Reply:
Will add

Reply:
Docs have this cool product dropdown thing:

![image](https://user-images.githubusercontent.com/8484333/197215299-358a7700-a368-4dee-9d0f-3b56e47d94d2.png)

Always thought it would be handy to copy that for "What feature are you having difficulty with?"

- `publish`
- `dev`
- `pages ...`
- `unstable_dev`